base_layer/wallet/src/types.rs

```diff
@@ -20,8 +20,13 @@
 // WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE
 // USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
+use tari_core::transactions::tari_amount::MicroTari;
 use tari_crypto::common::Blake256;
 
+/// The default fee per gram that the wallet will use to build transactions.
+/// TODO discuss what the default fee value should actually be
+pub const DEFAULT_FEE_PER_GRAM: MicroTari = MicroTari(25);
```

Comment:
Minimum fee in mobile wallet is set to 100.

Not sure if these should be in sync?

Reply:
This is fee_per_gram not absolute fee. This is also an arb selection and needs to be discussed as stated in the TODO